index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+    <script src='main.js'></script>
+</head>
+<body>
+    <h2>Hello Microverse!</h2>
```

Comment:
I think you should use the `h1` tag for the first header text to indicate the primary heading on a webpage which is also the main visible header on the page. Please remember that using the `h1` tag is also an important requirement for this project 😃 

![correct2](https://user-images.githubusercontent.com/20537712/232855599-750e20cf-7194-4b25-8352-84cf1282d447.png)